src/client/producer.rs

```diff
@@ -213,7 +213,7 @@ mod broadcast;
 #[derive(Debug, Error)]
 pub enum Error {
     #[error("Aggregator error: {0}")]
-    Aggregator(#[from] aggregator::Error),
+    Aggregator(#[from] Arc<aggregator::Error>),
```

Comment:
Same as below, we have `Arc` of `Box` here

Reply:
fixed